include/stl2/detail/concepts/core.hpp

```diff
@@ -69,6 +61,9 @@ STL2_OPEN_NAMESPACE {
 	template <class T, class... Args>
 	concept bool _OneOf = models::_OneOf<T, Args...>;
 
+	template <class T, class U>
+	concept bool _NotSameAs = __bool<!__is_same_as(__uncvref<T>, __uncvref<U>)>;
```

Comment:
Ditto weird `__bool`.

Reply:
Doubly so given that we don't need to defend internal concepts against subsumption.